presto-main/src/test/java/io/prestosql/operator/scalar/CustomFunctions.java

```diff
@@ -45,4 +45,11 @@ public static boolean customIsNullBigint(@SqlNullable @SqlType(StandardTypes.BIG
     {
         return value == null;
     }
+
+    @ScalarFunction(value = "identity.function", alias = "identity&function")
```

Comment:
what if you declare other method with name `identity%function` with different body? Which one will be called or what kind of error message will be raised?

Reply:
It might throw `Function already registered: `